nginx-php5/Dockerfile

```diff
@@ -1,12 +1,11 @@
-FROM unit9/base:latest
+FROM unit9/nginx:latest
 
 MAINTAINER Artiom Vasiliev <artiom@unit9.com>
 
 COPY ["php-fpm.rc", "/etc/service/php-fpm/run"]
-COPY ["nginx.rc", "/etc/service/nginx/run"]
 
 RUN apt-get update && \
-    apt-get install -y nginx php5 php5-common php5-curl php5-fpm php5-gd \
+    apt-get install -y php5 php5-common php5-curl php5-fpm php5-gd \
         php5-mysql php5-xmlrpc php5-xcache php5-memcache && \
     rm -rf /var/cache/apt /var/lib/apt/lists
 
```

Comment:
Line 14 (sorry github doesn't allow me to post a comment there), it looks like the nginx.conf is installed as the default site, and nginx.conf from base nginx doesn't pull it